proseco/tests/test_guide.py

```diff
@@ -286,3 +286,45 @@ def test_guides_include_exclude():
 
     assert np.all(guides['id'] == [9, 11, 2, 3, 4, 5, 6, 7])
     assert np.allclose(guides['mag'], [10.0, 12.0, 7.1, 7.2, 7.3, 7.4, 7.5, 7.6])
+
+
+def test_edge_star():
+    """
+    Add stars right at row and col max for 8" dither.
```

Comment:
This test is good but it seems like it does not actually test the bug fix related to get_ax_range.  That would happen with hot pixel imposters right?

Also in both this test and a synthetic hot pixel test, document the exact source of the magic numbers (e.g. 495.3).  I'm never sure whether it is best to have test values like this hardcoded, so it gives a visible regression if characteristics change, or have them computed in the test so tests continue passing.  If you don't care then I'd say the latter.
```
        row_max = CCD['row_max'] - (CCD['row_pad'] + CCD['window_pad'] + r_dith_pad)
        col_max = CCD['col_max'] - (CCD['col_pad'] + CCD['window_pad'] + c_dith_pad)
```

Also in both tests it would be best to use asymmetric dither, esp for imposters since that was the original point of the bug fix.

Reply:
Yeah, I was working on a new test of hot pix imposters Friday but didn't finish.  Will get it in.  And sure, documenting exact source of the numbers sounds good (I at least left a note about nominal dither being required for the test setup, but can also recalculate it directly here.  Figured it was unlikely we'd change CCD characteristics without needing to update all the tests anyway.).  For the edge case test, I don't think it matters if the test is done with asymmetric dither or not, but fine to do so.  For my specific bug also need to test with the "large" dither in each axis.